packages/babel-preset-env/package.json

```diff
@@ -11,7 +11,7 @@
   "repository": "https://github.com/babel/babel/tree/master/packages/babel-preset-env",
   "main": "lib/index.js",
   "dependencies": {
-    "@babel/compat-data": "^7.8.6",
+    "@babel/compat-data": "~7.8.6",
```

Comment:
`@babel/compat-data` is also depended by `@babel/helper-compilation-targets`. By doing so we risk having two versions of `@babel/compat-data` if one installs preset-env + helper-compilation-targets 7.8.0 while compat-data 7.9.0 has been published.

Can we fix this on the preset-env side? So it can work with higher `compat-data` as long as we would not drop entries in minor releases. We can introduce an extra unit test as a sync reminder.